numpy/add_newdocs.py

```diff
@@ -3680,7 +3680,7 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Examples
     --------
-    >>> x = np.random.randint(9, size=(3, 3))
+    >>> x = np.random.randint(low=9, size=(3, 3))
     >>> x
```

Comment:
Keep the example as it was. Add an example showing the function called with no args now that that's possible.